sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -330,39 +335,56 @@ void ins_int_propagate(struct Int32Vect3 *accel, float dt)
 
 static void baro_cb(uint8_t __attribute__((unused)) sender_id, float pressure)
 {
-  if (!ins_int.baro_initialized && pressure > 1e-7) {
-    // wait for a first positive value
-    ins_int.qfe = pressure;
-    ins_int.baro_initialized = true;
+  if (pressure < 1000)
+  {
+    // bad baro pressure, don't use
+    return;
   }
 
-  if (ins_int.baro_initialized) {
-    if (ins_int.vf_reset) {
-      ins_int.vf_reset = false;
-      ins_int.qfe = pressure;
-      vff_realign(0.);
-      ins_update_from_vff();
-    } else {
-      float baro_up = pprz_isa_height_of_pressure(pressure, ins_int.qfe);
+  if (!ins_int.baro_initialized && pressure > 1e-6) {
```

Comment:
So the 1000 there is actually up for discussion. I have seen some strange things here with the baro on the bebop where there are some really small values returned but is quite sporadic so was not able to log the actual value. What is a reasonable number to check for a bad measurement (instead of 1000 which is quite arbitrary).